getting-started-instructions.md

```diff
@@ -0,0 +1,50 @@
+# Setting up the GP2GP Adaptors within INT
+
+The below instructions assume you will be using a single message broker for all messages.
+
+### Setting up the MHS adaptor
+
+This is a __required__ prerequisite for both requesting and sending adaptors. 
+
+1. Identify a HSCN IP address(es) which can be used for the GP2GP process, and made available to the machine running the MHS Inbound.
+   This IP will be used by spine when initiating HTTPS requests into your environment.
+1. Request a new FQDN `<gp2gpname>.<suppliername>.thirdparty.nhs.uk` from NHS England DNS team and request it resolve to the HSCN IP
+   identified above. Include some indicator in the gp2gp name that this is a test/INT hostname, to differentiate it from the live hostname.
+   Example FQDN: `test-gp2gp.adrian.thirdparty.nhs.uk`.
+1. Create an Certificate Signing Request for the newly created FQDN, submit to the NHS ITOC requesting an endpoint for this.
+   During the creation of the CSR you'll be given a private key, copy this into the `MHS_SECRET_CLIENT_KEY` environment variable for Inbound/Outbound adaptors.
+1. ITOC will provide a certificate once this has been done, copy this into the `MHS_SECRET_CLIENT_CERT` environment variable for Inbound/Outbound adaptors.
+1. Request from ITOC that new GP2GP product be registered in SDS to your test GP Surgery using the same endpoint FQDN you used above.
+1. ITOC will provide a Party Key once this has been done, copy this into the `MHS_SECRET_PARTY_KEY` environment variable for Inbound/Outbound adaptors.
+
+### Requesting adaptor
+
+1. Create a Postgres database, and run the application schema migration script,
+   followed by with the SNOMED import script.
+1. Spin up the PS Adaptor facade and translator, specifying `PS_DB_URL` to point at the database,
+   and the broker related environment variables.
+1. For the translator container, specify the `MHS_BASE_URL` variable pointing to the MHS Oubound created above, and `PS_DAISY_CHAINING_ACTIVE` to "true".
```

Comment:
Typo on Outbound, and where does it point to?

Reply:
Typo fixed, but I'm not quite sure I understand the question.